scripts/release.sh

```diff
@@ -16,7 +16,7 @@ dockerhub_tag_exists() {
 
 # Get the tag at the HEAD
 get_tag() {
-  vers=`git tag --points-at HEAD`
+  vers=`git tag --points-at HEAD | tail -n 1`
```

Comment:
Btw this is how dataplane is doing it:
```suggestion
  vers=`git describe --exact-match 2>/dev/null || echo ""`
```